smart-contracts/dev-environments/remix/deploy-a-contract.md

```diff
@@ -1 +1,86 @@
-TODO
\ No newline at end of file
+---
+title: Deploy and Interact with Smart Contracts Using Remix IDE
+description: Learn how to deploy smart contracts to the Polkadot Hub network and interact with them using the Remix IDE and wallet providers, covering deployment and state.
+categories: Smart Contracts, Tooling
+---
+
+# Deploy Smart Contracts Using Remix IDE
+
+## Overview
+
+After compiling your smart contract in Remix IDE, the next step is to deploy it to the Polkadot Hub network. This guide will walk you through the deployment process using a wallet provider and show you how to interact with your deployed contracts directly from the Remix interface.
+
+## Prerequisites
+
+Before deploying your contract, ensure you have:
+
+- Completed the [Remix IDE setup](/smart-contracts/dev-environments/remix/get-started/){target=\_blank} and have a compiled contract ready
+- A compatible wallet extension installed (e.g., [MetaMask](https://metamask.io/){target=\_blank} or [Talisman](https://www.talisman.xyz/){target=\_blank})
+- Your wallet connected to the Polkadot Hub network. Check the [Connect to Polkadot](/smart-contracts/connect/){target=\_blank} guide for more information.
+- Test tokens in your wallet to cover deployment and transaction fees (available from the [Polkadot faucet](/smart-contracts/faucet/){target=\_blank})
+
+## Deploying Contracts
+
+1. To deploy your contract, you need to:
+
+    1. Navigate to the **Deploy & Run Transactions** tab.
+    2. Click the **Environment** dropdown.
+    3. Select **Browser Extension**.
+    4. Click on the **Injected Provider - MetaMask** option.
+
+2. Click the **Deploy** button and then click **Confirm** in the wallet popup.
```

Comment:
Sorry - just noticing this now. This is kind of a weird break up of the steps like if you follow all of the steps under 1 you still haven't deployed the contract, so to say "To deploy a contract" doesn't totally fit